Client/Frontend/Settings/ThemesSettingsViewController.swift

```diff
@@ -0,0 +1,152 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Static
+import BraveShared
+import Shared
+import BraveRewards
+
+class ThemesSettingsViewController: UITableViewController {
+    
+    var themeApplied: (() -> Void)?
+    var themeChanged: ((_ themeId: String?) -> Void)?
+    
+    private enum Sections: Int, CaseIterable { case themes, appearance }
+    
+    override func viewDidLoad() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "default")
+        
+        title = Strings.themesSettings
+    }
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        Sections.allCases.count
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let customThemesCount = customThemeIDs.count + 1
+        let defaultThemesCount = Theme.DefaultTheme.normalThemesOptions.count
+        
+        return section == Sections.themes.rawValue ? customThemesCount : defaultThemesCount
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        section == Sections.themes.rawValue ? Strings.themesSettings : Strings.themesDisplayBrightness
```

Comment:
Oh weird, I didn't realize normal functions had implicit `return`. Makes sense when I think about it though.